bin/kano-ui-autostart

```diff
@@ -114,6 +114,9 @@ if [ "$DESKTOP_RUN" -eq 1 ] ; then
 	sudo kano-updater first-boot
     fi
 
+    # Play a welcome sound before entering the Dashboard screen
+    aplay "/usr/share/kano-media/sounds/kano_boot_up.wav"
```

Comment:
Do we also want to use `-q` here or even the wrapper from [kano-toolset](https://github.com/KanoComputing/kano-toolset/blob/master/kano/utils/audio.py#L14)?